src/hotspot/jtreg/compiler/loopopts/TestDivZeroWithSplitIf.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
The test seems in wrong place. Should be in test/hotspot.

Reply:
Oh, right, this somehow got wrong when moving this patch from the jdk to the jdk16 repo. Fixed it.